lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1702,6 +1702,7 @@ public function get_collection_params() {
 		$params['filter'] = array(
 			'description'       => __( 'Use WP Query arguments to modify the response; private query vars require appropriate authorization.' ),
 		);
+		$params['filter[posts_per_page]'] = $params['per_page'];
```

Comment:
Do you mean like following?
```
$params['filter']['posts_per_page'] = $params['per_page'];
```
It will not work, because here is not supporting multidimensional arrays.
https://github.com/WP-API/WP-API/blob/develop/plugin.php#L284-L358

But I am not sure that it is best or not. :smile: